templates/focus-ios.xliff

```diff
@@ -28,6 +28,19 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Blockzilla/en.lproj/Intents.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="9.1" build-num="9B55"/>
+    </header>
+    <body>
+      <trans-unit id="Erase">
```

Comment:
Could we get a short comment here, just specifying if it's a noun or a verb? (I'm assuming it's a verb, but better to be extra clear in those cases ;) ).
Also, not sure this is a valid ID?